src/lib/knowledgeAI/extract.ts

```diff
@@ -111,12 +117,21 @@ export const extract = async (type: string, options: IExtractOptions) => {
 
     spinner.stop();
     if (content) {
-        await writeResultsToFile(options.outputFile, content);
+
+        // prepend ctxt version number
+        const cContent = "`version: 1`\n\n" + content;
```

Comment:
Just a thought, do you think we hardcode the `version: 1` text, I am thinking it is not configurable.
When we introduce a new version, do we always have to change this?

This comment can be completely irrelevant, though. Thanks!

Reply:
well this, is an npm package, we will need to do a new release anyways,